heltour/tournament/migrations/0002_registration_last_validation_try.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.20 on 2025-03-17 12:09
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tournament', '0001_squashed_0195'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='registration',
+            name='last_validation_try',
+            field=models.DateTimeField(default=django.utils.timezone.now),
```

Comment:
Does this mean they all start with `now` as the value when we start?

Reply:
yes, they will start with `now`. i guess we could pick any other value too, as long as it is not in the future.

Reply:
Yeah, I think it's fine?  Will this queue  ever be "complete"? or will there always be more to process?

Reply:
never complete, every 5 minutes it will try to validate a registration, if there is at least one that was not successfully validated yet (in any season that has open registrations). that is why i was considering filtering for a minimum time delta too, so that re-validation of the same reg is only attempted after, say, 12 hours.
in the extreme case of only one not successfully validated reg, we would currently re-validate that one every 5 minutes.
the only way it would probably ever be complete would be if there was not season with open registrations. which might happen now and then, but not all too often.

Reply:
Yeah, I like filtering for "hasn't been updated in 12 hours".